requirements/dev.in

```diff
@@ -6,4 +6,4 @@
 diff-cover                # Changeset diff test coverage
 edx-i18n-tools            # For i18n_tool dummy
 django-debug-toolbar      # For debugging Django
-#transifex-client         # For managing translations
+#transifex-client         # For managing translations
```

Comment:
It looks like the newline is missing. Could you add it back in, please?

Reply:
I'm having trouble interpreting this, are you asking for me to add a newline before, after, or in place of the line highlighted in green?

Reply:
Fixed: Added the newline at the end